Update environments-overview.md (#1896)
spryker/spryker-docs: docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md

{% info_block infoBox "Note" %}

> The contents of contracts may vary, and this document describes the usual setup. If you are using a different setup, ask your project manager or product owner for a more detailed explanation.

Reviewer: ```suggestion
The contents of contracts may vary, and this document only describes the default setup. If you are using a different setup, ask your project manager or product owner for a more detailed explanation.
```

> Please note that your non-production environments do not scale automatically. If you want to request to increase their scaling, please let us know via this dedicated form in our support portal: [Upscale Request](https://support.spryker.com/s/hosting-change-requests/environment-upscaling)

Reviewer: ```suggestion
Please note that your non-production environments do not scale automatically. If you want to increase their scaling, please let us know by requesting through this dedicated form in our support portal: [Upscale Request](https://support.spryker.com/s/hosting-change-requests/environment-upscaling)
```

{% endinfo_block %}

## Basic concept

> Spryker offers you environments that serve different purposes. These environments are non-production and production environments. We will configure these environments to be as similar as possible to make sure that the code you deploy on one platform behaves the same way on the others. You can help us keep your environments consistent by taking this into consideration when sending us change requests.

Reviewer: ```suggestion
Spryker offers you environments that serve different purposes. These environments are non-production and production environments. We can configure these environments to be as similar as possible to make sure that the code you deploy on one platform behaves the same way on the others. You can help us keep your environments consistent by taking this into consideration when sending us change requests.
```

### Developer environment (DEV) (optional)

> A Dev environment is a non production environment that is generally used to test code that just left your development team or is used in the development process directly through continuous deployment. It is not designed to handle load tests and large datasets. The development environment does not provide autoscaling and cannot be increased.

Reviewer: ```suggestion
A Dev environment is a non production environment that is usually used to test code that just left your development team, or is used in the development process directly through continuous deployment. It is not designed to handle load tests and large datasets. The development environment does not provide autoscaling and cannot be increased.
```

### Staging (STAGE)

> This environment is a non production environment that is generally used to host a snapshot of the Dev environment or a stable version of your code that is deployed to production eventually. You can use your staging environment to do performance testing to see how your code behaves in your production environment. However, staging is not of the same size as production, and a subset of data and expected traffic must be considered while running the test. When demoing your shop, this environment is used regularly. The staging environment provides limited autoscaling capabilities and can only be scale dup upon request.

Reviewer: ```suggestion
This environment is a non production environment that is usually used to host a snapshot of the Dev environment, or a stable version of your code that is deployed to production eventually. You can use your staging environment to do performance testing to see how your code behaves in a production environment. However, staging is not of the same size as production, and a subset of data and expected traffic must be considered while running the test. When demoing your shop, this environment is used regularly. The staging environment provides limited autoscaling capabilities and can only be scale dup upon request.
```
